src/lua/api-gateway/tracking/validator/rewritingRulesValidator.lua

```diff
@@ -32,6 +32,16 @@ local _M = BaseValidator:new()
 
 local function getBackendFromRewriteRule( rewrite_rule )
     local backend = rewrite_rule.meta or nil
+    local variableManager = ngx.apiGateway.tracking.variableManager
+    if ( variableManager == nil ) then
+        ngx.log(ngx.WARN, "Please initialize RequestVariableManager before calling this method")
```

Comment:
if `variableManager` is `nil` this method yields a runtime exception at line 43 bellow.